components/asset/introduction.rst

```diff
@@ -180,9 +192,12 @@ context of the current request::
     // result: /somewhere/static/images/logo.png?v1
 
 When the request context is set (via an optional third argument), in addition to
-the configured base path, ``PathPackage`` also prepends the current request base
-URL (``/somewhere/`` in this example) to assets. This allows your website to be
-hosted anywhere under the web server root directory.
+the configured base path (i.e. ``/static/images``), ``PathPackage`` also
+prepends the current request base URL. So, for example, if your entire site
```

Comment:
I find this first sentence really hard to understand. What about *"Now the request context is set, the ``PathPackage`` prepends the current request base URL. So, for example, if your entire site is hosted under the ``/somewhere`` directory in your web server root directory and the configured base path is ``/static/images``, all paths will be prefixed with ``/somewhere/static/images``."*